client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -391,7 +394,7 @@ public <K, C> ShuffleReader<K, C> getReaderImpl(
     final int indexReadLimit = sparkConf.getInt(RssSparkConfig.RSS_INDEX_READ_LIMIT,
         RssSparkConfig.RSS_INDEX_READ_LIMIT_DEFAULT_VALUE);
     RssShuffleHandle rssShuffleHandle = (RssShuffleHandle) handle;
-    final String shuffleRemoteStoragePath = rssShuffleHandle.getRemoteStorage();
+    final String shuffleRemoteStoragePath = rssShuffleHandle.getRemoteStorage().getUriStr();
```

Comment:
```suggestion
    final String shuffleRemoteStoragePath = rssShuffleHandle.getRemoteStorage().getPath();
```

Reply:
I forgot to update spark3 related codes will fix it, thanks.